Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
input
```php

    /**/
    public function method444g()
    {
    }
```

output
```php
    /**/
    public function method444g()
    {
    }
```

I don't see the problem, you want the blank line back? Then this PR is not needed in the first place ^-^